### ISSUE TITLE
Projet api get projects(projectFilter,itemsPerPage)

### ISSUE THREAD

**Issue by jflefebvre06** [gitlab4j/gitlab4j-api#605]
Hello
Could you please add search options search_namespaces and simple to projectFilter
 https://docs.gitlab.com/ee/api/projects.html#list-all-projects

**Comment by gmessner** [comment]
@jflefebvre06 ```simple``` is currently available in ```ProjctFilter``` but I will add the following fields that are new sine it was originally implemented:
```
search_namespaces
id_after
id_before
last_activity_after
last_activity_before
repository_storage
```
The next release is schedule for 8/15/2020

**Comment by gmessner** [comment]
@jflefebvre06 This has been released in 4.15.6, please verify and close this issue when you have the time.

**Comment by jflefebvre06** [comment]
Thank you
